### PR TITLE
Remove getVibrate / getSoundEnabled apis and logics

### PR DESCRIPTION
**Pull request by Jeasmine**
* Remove getVibrate and enableVibrate apis
* Remove getSoundEnabled and enableSound apis
* Remove logics using them

This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-android-sdk/1188)

### CODE REVIEW

File: OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java [hunk 1]
```diff
@@ -240,28 +240,9 @@ private static void setAlertnessOptions(JSONObject fcmJson, NotificationCompat.B
          } catch (Throwable t) {
             notificationDefaults |= Notification.DEFAULT_LIGHTS;
          } // Can throw if an old android support lib is used or parse error.
-      }
-      else
+      } else {
          notificationDefaults |= Notification.DEFAULT_LIGHTS;
-
-      if (OneSignal.getVibrate() && fcmJson.optInt("vib", 1) == 1) {
```
Comment (jkasten2) [inline OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java:247]: We just want to remove the `OneSignal.getVibrate()` call here. As a backwards compatibility for pre-O devices the OneSignal backend will still send "vib" depending if the Notification Category has this set.

File: OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java [hunk 2]
```diff
@@ -929,13 +910,6 @@ private static int getDrawableId(String name) {
       return contextResources.getIdentifier(name, "drawable", packageName);
    }
 
-   private static boolean isSoundEnabled(JSONObject fcmJson) {
```
Comment (jkasten2) [inline OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java:932]: Keep this method, just remove the `OneSignal.getSoundEnabled();` check. Same reason noted above about the Android Notification Categories

**Review by jkasten2 — CHANGES_REQUESTED** [review]

**Review by jkasten2 — APPROVED** [review]

**Comment by jkasten2** [comment]
Flaky tests notes. One failed on `testAfterLastInAppTimeIsDisplayed` and the other on `testNotificationWillShowInForegroundHandler_notCallCompleteShowsNotificationAfterTimeout`

**Comment by Jeasmine** [comment]
> Flaky tests notes. One failed on `testAfterLastInAppTimeIsDisplayed` and the other on `testNotificationWillShowInForegroundHandler_notCallCompleteShowsNotificationAfterTimeout`

noted!

**Pull request merged**